.github/workflows/build.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
         with:
           gradle-home-cache-cleanup: true
 
-      - run: ./gradlew build :samples:jest:run :mosaic-runtime:dokkaHtml
+      - run: ./gradlew build :samples:jest:run :mosaic-runtime:dokkaHtml apiCheck
```

Comment:
This isn't needed, as it should already run as part of `check` which is run as part of `build`

Reply:
Great, I didn't know there was such a task.